packages/core/src/dropdown/dropdown.stories.ts

```diff
@@ -0,0 +1,591 @@
+        anchor-align="end"
+        aria-labelledby="anchored-dropdown-title"
+        id="demo-anchored-dd"
+      >
+        <div cds-layout="vertical gap:lg p:lg align:stretch">
+          <h3 cds-first-focus cds-text="section" id="anchored-dropdown-title">Anchored Dropdown</h3>
+          <div cds-layout="vertical gap:lg">
+            <p cds-text="body">A dropdown can be anchored to any element on the page.</p>
+            <p cds-text="body">It doesn't have to be anchored to the element that the user clicks on to show it.</p>
+          </div>
+          <cds-button style="max-height: 36px" type="button" @click=${() => this.popup.closeOverlay('custom')}
+            >Close Dropdown</cds-button
+          >
+        </div>
+        <cds-internal-pointer type="default"></cds-internal-pointer>
+      </cds-dropdown>`;
+  }
+}
+
+@customElement('demo-dropdown-orientation')
+class DemoDropdownOrientation extends DemoDropdown {
+  @state({ type: String }) orientationPref = 'default';
+
+  updateOrientation(newPref: string) {
+    this.orientationPref = newPref;
+  }
+
+  private get myOrientationPref() {
+    switch (this.orientationPref) {
+      case 'vertical':
+        return 'not:left not:right';
+      case 'horizontal':
+        return 'not:top not:bottom';
+      case 'top-left':
+        return 'top left';
+      case 'bottom':
+        return 'only:bottom';
+      default:
+        return '';
+    }
+  }
+
+  private get myOrientationDisplay() {
+    switch (this.orientationPref) {
+      case 'vertical':
+        return 'The dropdown will only try positioning the dropdown to above or below the anchor if the orientation preference is set to "not:left not:right".';
+      case 'horizontal':
+        return 'The dropdown will only try positioning the dropdown to the left or right of the anchor if the orientation preference is set to "not:top not:bottom".';
+      case 'top-left':
+        return 'Setting the preferences to "top left" will have the dropdown try to position itself first above the anchor then to the left of it. Then it will try the remaining positions of bottom and right.';
+      case 'bottom':
+        return 'The dropdown\'s orientation is set to "only:bottom". This means the dropdown will only position itself below the anchor. If it cannot fit, it will show a responsive dropdown.';
+      default:
+        return 'Default preferences are bottom, right, left, and top. The dropdown will try to position itself in relation to the anchor in that order.';
+    }
+  }
+
+  private orientationPrefCode() {
+    if (this.myOrientationPref !== '') {
+      return html`<p cds-text="body">
+        Orientation preference is set to <span cds-text="code">${this.myOrientationPref}</span>.
+      </p>`;
+    } else {
+      return '';
+    }
+  }
+
+  render() {
+    return html` <div cds-layout="vertical gap:lg">
+        <div>
+          <cds-radio-group layout="horizontal-inline">
+            <label>Dropdown Preferences:</label>
```

Comment:
This demo is better when viewed on the main page than in isolation. In isolation, it doesn't have enough space around it to show what it's doing. But it does demo the fallback-to-responsive behavior pretty well. To view it in isolation, we would need to super-increase the space around the trigger button. This would make it really big on the main page.

So if we want to go that route, we should remove this demo from the main storybook page. But if we keep it on the main storybook page, we shouldn't add a bunch of space around it. 

Which path do you prefer?

Reply:
What you have makes sense, the main page working seems ideal over the isolated